pkg/operator/sync.go

```diff
@@ -19,26 +19,33 @@ import (
 	"github.com/openshift/machine-config-operator/pkg/version"
 )
 
-type syncFunc func(config renderConfig) error
+type syncFunc struct {
+	name string
+	fn   func(config renderConfig) error
+}
 
 func (optr *Operator) syncAll(rconfig renderConfig) error {
 	// syncFuncs is the list of sync functions that are executed in order.
 	// any error marks sync as failure but continues to next syncFunc
-	syncFuncs := []syncFunc{
-		optr.syncMachineConfigPools,
-		optr.syncMachineConfigController,
-		optr.syncMachineConfigServer,
-		optr.syncMachineConfigDaemon,
-		optr.syncRequiredMachineConfigPools,
+	var syncFuncs = []syncFunc{
+		{ "pools", optr.syncMachineConfigPools },
+		{ "mcs", optr.syncMachineConfigServer },
+		{ "mcd", optr.syncMachineConfigDaemon },
+		{ "mcc", optr.syncMachineConfigController },
+		{ "required-pools", optr.syncRequiredMachineConfigPools },
 	}
 
 	if err := optr.syncProgressingStatus(); err != nil {
 		return fmt.Errorf("error syncing progressing status: %v", err)
 	}
 
 	var errs []error
-	for _, f := range syncFuncs {
-		errs = append(errs, f(rconfig))
+	for _, sf := range syncFuncs {
+		startTime := time.Now()
+		errs = append(errs, sf.fn(rconfig))
+		if optr.inClusterBringup {
+			glog.Infof("[init mode] synced %s in %v", sf.name, time.Since(startTime))
```

Comment:
Oh this is neat!

Reply:
why report only on cluster bringup ?

Reply:
Currently there are some `.V(4)` level logs in `sync()`.  It wasn't clear to me at what cadence we'd end up logging.  Glancing at the operator...it watches all daemonsets in our namespace, so we'll do a sync every time e.g. a node updates and the MCD state changes right?

I guess we can do more sync logging if we more consistently add some rate-limiting/delays like https://github.com/openshift/machine-config-operator/pull/337 ?